From 75bdcd52ddcc669ef62a891c7b1a6c4306259b16 Mon Sep 17 00:00:00 2001
From: Diego Wilson <diegowilson@google.com>
Date: Fri, 24 Feb 2017 17:07:28 -0800
Subject: [PATCH 06/11] msm8974: Move device dependent modules to /vendor
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change-Id: I929bc7c64e86bc890ad2c0e71d0c9d0fba6dfc11
See I8229a6e4a9512a03cbc6c9522bb1dc0ce988d1c9
Also added the libloc_api-rpc/-50001 MKs
Signed-off-by: D. Andrei Măceș <Andrei.Maces@alumni.nd.edu>
---
 msm8974/core/Android.mk                                         | 1 +
 .../loc_api/libloc_api-rpc-50001/libloc_api-rpc-glue/Android.mk | 1 +
 .../loc_api/libloc_api-rpc-50001/libloc_api-rpc-stub/Android.mk | 1 +
 msm8974/loc_api/libloc_api-rpc/Android.mk                       | 1 +
 msm8974/loc_api/libloc_api_50001/Android.mk                     | 2 ++
 msm8974/utils/Android.mk                                        | 2 ++
 6 files changed, 8 insertions(+)

diff --git a/msm8974/core/Android.mk b/msm8974/core/Android.mk
index 8ca6560..b7fb8c3 100644
--- a/msm8974/core/Android.mk
+++ b/msm8974/core/Android.mk
@@ -7,6 +7,7 @@ include $(CLEAR_VARS)
 
 LOCAL_MODULE := libloc_core
 LOCAL_MODULE_OWNER := qcom
+LOCAL_PROPRIETARY_MODULE := true
 
 LOCAL_MODULE_TAGS := optional
 
diff --git a/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-glue/Android.mk b/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-glue/Android.mk
index 3d09024..ce55784 100644
--- a/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-glue/Android.mk
+++ b/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-glue/Android.mk
@@ -52,6 +52,7 @@ LOCAL_COPY_HEADERS:= \
 
 LOCAL_MODULE:= libloc_api-rpc-qc
 LOCAL_MODULE_OWNER := qcom
+LOCAL_PROPRIETARY_MODULE := true
 
 LOCAL_MODULE_TAGS := optional
 
diff --git a/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-stub/Android.mk b/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-stub/Android.mk
index 1f542e4..4b9de62 100644
--- a/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-stub/Android.mk
+++ b/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-stub/Android.mk
@@ -8,6 +8,7 @@ LOC_RPCGEN_APIS_PATH_FL := ../../../../../$(TARGET_OUT_INTERMEDIATES)/loc_api/li
 
 LOCAL_MODULE := libloc_api_rpcgen
 LOCAL_MODULE_OWNER := qcom
+LOCAL_PROPRIETARY_MODULE := true
 
 LOCAL_SHARED_LIBRARIES := \
     librpc \
diff --git a/msm8974/loc_api/libloc_api-rpc/Android.mk b/msm8974/loc_api/libloc_api-rpc/Android.mk
index 06e4694..e438544 100644
--- a/msm8974/loc_api/libloc_api-rpc/Android.mk
+++ b/msm8974/loc_api/libloc_api-rpc/Android.mk
@@ -54,6 +54,7 @@ LOCAL_C_INCLUDES:= \
 
 LOCAL_MODULE:= libloc_api-rpc
 LOCAL_MODULE_OWNER := qcom
+LOCAL_PROPRIETARY_MODULE := true
 
 include $(BUILD_STATIC_LIBRARY)
 
diff --git a/msm8974/loc_api/libloc_api_50001/Android.mk b/msm8974/loc_api/libloc_api_50001/Android.mk
index e75db11..55e83cc 100644
--- a/msm8974/loc_api/libloc_api_50001/Android.mk
+++ b/msm8974/loc_api/libloc_api_50001/Android.mk
@@ -7,6 +7,7 @@ include $(CLEAR_VARS)
 
 LOCAL_MODULE := libloc_eng
 LOCAL_MODULE_OWNER := qcom
+LOCAL_PROPRIETARY_MODULE := true
 
 LOCAL_MODULE_TAGS := optional
 
@@ -60,6 +61,7 @@ include $(CLEAR_VARS)
 
 LOCAL_MODULE := gps.$(BOARD_VENDOR_QCOM_GPS_LOC_API_HARDWARE)
 LOCAL_MODULE_OWNER := qcom
+LOCAL_PROPRIETARY_MODULE := true
 
 LOCAL_MODULE_TAGS := optional
 
diff --git a/msm8974/utils/Android.mk b/msm8974/utils/Android.mk
index 955e20d..1135da9 100644
--- a/msm8974/utils/Android.mk
+++ b/msm8974/utils/Android.mk
@@ -51,6 +51,8 @@ LOCAL_COPY_HEADERS:= \
    loc_misc_utils.h
 
 LOCAL_MODULE := libgps.utils
+LOCAL_MODULE_OWNER := qcom
+LOCAL_PROPRIETARY_MODULE := true
 
 LOCAL_MODULE_TAGS := optional
 
-- 
2.17.1

